src/github.com/getlantern/yamlconf/yamlconf.go

```diff
@@ -103,6 +103,14 @@ type Manager struct {
 	// example for fetching config updates from a remote server.
 	CustomPoll func(currentCfg Config) (mutate func(cfg Config) error, waitTime time.Duration, err error)
 
+	// Obfuscate: if true, the on-disk version of the config will be obfuscated by
+	// encrypting it using AES-128 encryption.
```

Comment:
Why this choice? Since it won't provide any kind of security, rot13+zip could've been sufficient. Not that important, but I wonder if a more lightweight solution could be possible.